flax/__init__.py

```diff
@@ -33,3 +33,4 @@
 # Allow `import flax`; `flax.nn.[...]`, and the same for `flax.optim.[...]`
 from . import nn
 from . import optim
+from . import training
```

Comment:
Why is this import here?

Reply:
I was exposing it to run the imagenet example, but retesting now I realized it is unnecessary. I will remove it, sorry for the confusion